cmd/cluster-svcat-controller-manager-remover/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	operatorapiv1 "github.com/openshift/api/operator/v1"
+	operatorclient "github.com/openshift/client-go/operator/clientset/versioned"
+	log "github.com/sirupsen/logrus"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+var targetNamespaceName = "openshift-service-catalog-controller-manager-operator"
+
+func createClientConfigFromFile(configPath string) (*rest.Config, error) {
+	clientConfig, err := clientcmd.LoadFromFile(configPath)
+	if err != nil {
+		return nil, err
+	}
+
+	config, err := clientcmd.NewDefaultClientConfig(*clientConfig, &clientcmd.ConfigOverrides{}).ClientConfig()
+	if err != nil {
+		return nil, err
+	}
+	return config, nil
+}
+
+func main() {
+	log.Info("Starting openshift-service-catalog-controller-manager-remover job")
+
+	clientConfig, err := rest.InClusterConfig()
+	if err != nil {
+		clientConfig, err = createClientConfigFromFile(homedir.HomeDir() + "/.kube/config")
+		if err != nil {
+			//log.Error("Failed to create LocalClientSet")
+			//return nil, err
+			log.Error("Failed to create LocalClientSet")
+			panic(err.Error())
+		}
+	}
+
+	kubeClient, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	operatorClient, err := operatorclient.NewForConfig(clientConfig)
+	if err != nil {
+		log.Errorf("problem getting operator client, error %v", err)
+	}
+	operatorConfigClient := operatorClient.OperatorV1()
+	operatorConfig, err := operatorConfigClient.ServiceCatalogControllerManagers().Get("cluster", metav1.GetOptions{})
+	if err != nil {
+		log.Errorf("problem getting ServiceCatalogControllerManage CR, error %v", err)
+	}
+
+	switch operatorConfig.Spec.ManagementState {
+	case operatorapiv1.Managed:
+		log.Warning("We found a cluster-svcat-controller-manager-operator in Managed state. Aborting")
+		break
```

Comment:
Nit: No need to `break` since that's implicit in Go.

Reply:
Removed the breaks, that's not really a nit IMO.